portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path

```diff
@@ -23,6 +23,16 @@
 	<td>//option[contains(text(),'Stage')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>AUDIT_GRAPH_CONFIGURATION_LEFT_ARROW_BUTTON</td>
+	<td>//button[contains(@class, 'move-left')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>AUDIT_GRAPH_CONFIGURATION_RIGHT_ARROW_BUTTON</td>
+	<td>//button[contains(@class, 'move-right')]</td>
+	<td></td>
+</tr>
 <tr>
```

Comment:
Aren't this buttons already created in the Button.path? Can you use a generic button path?

Reply:
I couldn't find these button there, I could only find an arrow down. And the issue with a generic path is that they all require 'key_text', and those two buttons don't have any.